WordPress/Classes/ViewRelated/Basic Profile/BasicUserProfileViewController.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+final class BasicUserProfileViewController: UIViewController, DrawerPresentable, StoryboardLoadable {
+    static var defaultStoryboardName: String = "BasicUserProfile"
+
+    var expandedHeight: DrawerHeight = DrawerHeight.contentHeight(0)
+
+    let allowsUserTransition = false
+
+    @IBOutlet private weak var gravatarImageView: CircularImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var usernameLabel: UILabel!
+    @IBOutlet private weak var locationLabel: UILabel!
+    @IBOutlet private weak var aboutMeLabel: UILabel!
+    @IBOutlet private weak var sitesTableView: UITableView!
+
+    var viewModel: BasicUserProfileViewModel!
+
+    @objc class func present(context: UIViewController, view: UIView, email: String?) {
+        guard let email = email else {
+            return
+        }
+        let controller = BasicUserProfileViewController.loadFromStoryboard()
+        controller.viewModel = BasicUserProfileViewModel(email: email)
+
+        let bottomSheet = BottomSheetViewController(childViewController: controller)
+        bottomSheet.show(from: context, sourceView: view, arrowDirections: .up)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // TODO: Ghost animation
+        // TODO: Accessibility
+        // TODO: Dark mode
+        // TODO: Tracking
+        // TODO: Styling
```

Comment:
unless these TODOs represent missing code in this specific location, I would not leave them here. Also, to easily identify them and avoid forgetting them in the code, I usually label them more specifically, so you can search them easily (e.g. TODO - BASICPROFILES: - ...)

Reply:
Done.